externs/closure-compiler.js

```diff
@@ -15,3 +15,13 @@
  * @see http://www.w3.org/TR/pointerevents/#the-touch-action-css-property
  */
 CSSProperties.prototype.touchAction;
+
+/**
+ * @type {number}
+ */
+WheelEvent.DOM_DELTA_LINE;
```

Comment:
FYI I've added these to the closure-compiler externs (see https://github.com/google/closure-compiler/pull/1478)